docs/src/includes/_enums.md

```diff
@@ -652,6 +668,22 @@ Name | Data | Description
 [VISIBLE](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=PAUSEUI_VISIBILITY.VISIBLE) | 2 | 
 [SLIDING_UP](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=PAUSEUI_VISIBILITY.SLIDING_UP) | 3 | 
 
+## PAUSE_FLAG
+
+
+> Search script examples for [PAUSE_FLAG](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=PAUSE_FLAG)
+
+
+
+Name | Data | Description
+---- | ---- | -----------
+[MENU](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=PAUSE_FLAG.MENU) | 1 | 
+[FADE](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=PAUSE_FLAG.FADE) | 2 | 
+[CUTSCENE](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=PAUSE_FLAG.CUTSCENE) | 3 | 
+[FLAG4](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=PAUSE_FLAG.FLAG4) | 4 | 
+[FLAG5](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=PAUSE_FLAG.FLAG5) | 5 | 
+[ANKH](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=PAUSE_FLAG.ANKH) | 6 | 
+
```

Comment:
Are we really going to double every enum?

Reply:
I don't really like it either but people have demonstrated that they don't understand bits and we started numbering these like this for some reason ages ago. Have a better idea? At least we'd need a new function to set and clear these values with a bitmask, I don't even want to do that |& crap.

Reply:
Technically you can make function in lua that will return the "index/flag number", not sure which one is better as both just complicate this stuff even more in my opinion, i can imagine someone just being more confused seeing two enum for one thing

Reply:
People were getting confused because "there's no way" to get the flag number from these bitmasks. Whatever, I removed PAUSE_FLAG and added a function to set bitmasks and you just have to know what you are doing.